src/ensembl/src/content/app/species/components/species-sidebar/SpeciesSidebar.tsx

```diff
@@ -0,0 +1,171 @@
+      props.fetchSidebarPayload();
+    }
+  }, [props.sidebarPayload, props.activeGenomeId]);
+
+  if (!props.sidebarPayload) {
+    return <div>No data to display</div>;
+  }
+  const payload = props.sidebarPayload;
+
+  type AnnotationEntry = {
+    label: string;
+    value?: string | null;
+    url?: string;
+  };
+  const assemblyDate = new Date(payload.assembly_date);
+  const formattedAssemblyDate = Intl.DateTimeFormat('en-GB', {
+    month: 'short',
+    year: 'numeric'
+  }).format(assemblyDate);
+  const annotationEntriesMap: AnnotationEntry[][] = [
+    [
+      {
+        label: 'Provider',
+        value: payload.annotation_provider.name,
+        url: payload.annotation_provider.url
+      },
+      { label: 'Method', value: payload.annotation_method }
+    ],
+    [
+      { label: 'Assembly date', value: formattedAssemblyDate },
+      { label: 'Gencode version', value: payload.gencode_version }
+    ],
+    [
+      { label: 'Database version', value: payload.database_version },
+      { label: 'Taxonomy ID', value: payload.taxonomy_id }
+    ]
+  ];
+
+  return (
+    <div>
+      <div className={styles.speciesDetails}>
+        <span className={styles.common_name}>{payload.common_name}</span>
+        <span className={styles.scientific_name}>
+          {payload.scientific_name}
+        </span>
+      </div>
+
+      {payload.strain && (
+        <div className={styles.strainDetails}>
+          <span className={styles.strain_type}>{payload.strain.type}</span>
+          <span className={styles.strain_value}>{payload.strain.value}</span>
+        </div>
+      )}
+
+      <div className={styles.sectionHead}>Assembly</div>
+      <div className={styles.assemblyDetails}>
+        <div className={styles.assemblyName}>{payload.assembly_name}</div>
+
+        <div className={styles.assemblySource}>
+          <ExternalReference
+            label={payload.assembly_provider.name}
+            linkText={payload.id}
+            to={payload.assembly_provider.url}
+          />
+        </div>
+        <div className={styles.standardLabelValue}>
+          <div className={styles.label}>Assembly level</div>
+          <div className={styles.boldValue}>{payload.assembly_level}</div>
+        </div>
+      </div>
+
+      <div className={styles.sectionHead}>Annotation</div>
+      <div className={styles.annotationDetails}>
+        {annotationEntriesMap.map((entries, group_index) => {
+          return (
+            <div key={group_index}>
+              {entries.map((entry, entry_index) => {
+                const labelClassNames = classNames(styles.label, {
+                  [styles.labelShort]: group_index === 0
+                });
+
+                return entry.value ? (
+                  <div key={entry_index} className={styles.standardLabelValue}>
+                    <div className={labelClassNames}>{entry.label}</div>
+                    <div className={styles.value}>
+                      {entry.url ? (
```

Comment:
We should either skip external reference if the provider is Ensembl, or link to some kind of about page.

Reply:
I've removed the link and just displaying Ensembl.